### PR TITLE
Feature/feature toggeling

### PR DESCRIPTION
**Pull request by manuwei**
* add config-server service and default values to enable feature toggling
* add warrany (Haftungsausschlus)

### CODE REVIEW

File: config/default-config.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "features": {
+    "designModel": false,
+    "patternCandidate": false,
+    "patternViews": false,
+    "issue": false,
+    "showSettings": false,
```
[the remark this answers is not in the document]
Comment (manuwei) [reply config/default-config.json:7]: this boolean is set to true if the settings can be changed  - do you suggest changing the default value to false?

[the remark this answers is not in the document]
Comment (salmma) [reply config/default-config.json:7]: ah ok, then it is already ok :)

File: docker-compose.yml [hunk 2]
```diff
@@ -25,8 +25,35 @@ services:
...
+      ETCD_ADVERTISE_CLIENT_URLS: "http://config-server:2379"
+      ETCD_LISTEN_CLIENT_URLS: "http://0.0.0.0:2379"
+      ETCDCTL_API: 3
+    ports:
+      - "2379:2379"
+    networks:
+      - default
+  # used by the pattern atlas to render latex
+  latex-renderer:
+    image: planqk/latex-renderer:v1.0.1
```
Comment (salmma) [inline docker-compose.yml:54]: Is this on purpose not latest? :)

**Review by salmma — APPROVED** [review]
LGTM

**Pull request merged**